contactSalon.css

```diff
@@ -0,0 +1,85 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+header{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 50%, 50%;
+  position: relative;
+  height: 120px;
+  background-color: #519194;
+}
+
+.logo{
+  width: 250px;
+  position: absolute;
+  bottom: -12px;
+  left: 13px;
```

Comment:
or just move this in more. like `left:100px;`.
plenty of options. pick what works!